Students/Ghiurutan V. Alexandru/Polynomial/src/Polynomial.java

```diff
@@ -0,0 +1,48 @@
+import java.text.DecimalFormat;
+
+public class Polynomial {
+	private double[] coeffs;
+
+	public Polynomial(double... coeffs) {
+		this.coeffs = coeffs;
+	}
+
+	public double[] getPolynomial() {
+		return coeffs;
```

Comment:
You can do this by either deep-cloning it here, or "manually" in any other place you call this method (it depends on what you want to do with this method)